sensitive-terms.txt

```diff
@@ -0,0 +1,1933 @@
+bitchin
+bitchin'
+bitching
+bitchlike
+bitchness
+bitchs
+bitchslap
+bitchtits
+bitchy
+bitchy ass
+bizzach
+bj
+black cock
+blacky
+blonde action
+blonde on blonde action
+blow a load
+blow job
+blow your load
+blowjob
+blue waffle
+blumpkin
+blumpkin boy
+boiolas
+boll0ck
+bollock
+bollockchops
+bollocks
+bollocktician
+bollox
+bondage
+boner
+bonk
+bonk juice
+boob
+boobs
+booty
+booty call
+breast
+breasticles
+breasts
+brown showers
+brunette action
+bugger
+bugger off
+bukkake
+bull shit
+bulldyke
+bullet vibe
+bullshi
+bullshit
+bullshite
+bum
+bum-bandits
+bum-driller
+bumhole
+bung
+bung hole
+bunghole
+busty
+butt
+butt fuck
+butt fucker
+butt hole
+butt pirate
+butt-fucker
+butt-fuckers
+butt-fucking-bandit
+butt-munchers
+buttcheeks
+buttfuck
+buttfucker
+buttfuckers
+butthole
+butthole boy
+bwc
+c*nt
+c*ntface
+c*nts
+c*nty
+c00n
+c00nies
+c0ck
+c0ckhead
+c0cks
+c0cksuccer
+c0cksucked
+c0cksucker
+c0cksuckers
+c0x
+c0xux0r
+caaak
+caca
+cack
+cagone
+cajones
+camel jockey
+camel toe
+cameltoe
+camgirl
+camslut
+camwhore
+carpet muncher
+carpetmuncher
+carpetmunchers
+carpetmunching
+cawk
+cawk muncher
+cawkmuncher
+cawks
+cawksucker
+ch1nk
+chesticles
+china virus
+china-virus
+chinavirus
+chinese-virus
+chingchong
+chink
+chinky
+choad
+choad nectar
+chocha
+chocolate rosebuds
+choke the chicken
+cholo
+cialis
+circlejerk
+cleveland steamer
+clit
+clitlickers
+clitoris
+closet fuckhead
+clover clamps
+clusterfuck
+clusterfucked
+clusterfucker
+clusterfucking
+clusterfucks
+cobb knobbler
+cock
+cock droplets
+cock head
+cock jockey
+cock knocker
+cock licker
+cock munch
+cock rider
+cock smoker
+cock snot
+cock suck
+cock sucker
+cock sucking
+cock sucking boiolas
+cock sucking nob jokeys
+cock tease
+cock-face
+cock-gobbler
+cock-head
+cock-sucker
+cock-suckers
+cockboy
+cockeater
+cockface
+cockgobbler
+cockh3ad
+cockhead
+cockheads
+cockhed
+cockless
+cocklicker
+cockmonger
+cockmunch
+cockmuncher
+cocks
+cockshit
+cockskin
+cocksmoker
+cocksucc
+cocksuccas
+cocksuccer
+cocksuccers
+cocksuck
+cocksuckas
+cocksucked
+cocksucker
+cocksuckers
+cocksucking
+cocksucking mothafuckers
+cocksucks
+cocksuk
+cocksuka
+cocksukka
+cocsuck
+cojones
+cokmuncher
+coksucka
+coolie
+coon
+coon1es
+cooni3s
+coonie
+coonies
+coons
+coot
+coot coot
+cooter
+cooter shooting
+cooterpuffing
+cooties
+coprolagnia
+coprophilia
+cotton picker
+crazy mofos
+crazy sob
+creampie
+crotch
+crotch fiddler
+crotchy
+crow
+cuksuker
+cuksukka
+cum
+cum goon
+cumball
+cumdumpster
+cumming
+cums
+cumshot
+cumshots
+cumslut
+cumz
+cunnilingus
+cunt
+cunt ass
+cunt fart
+cunt lick
+cunt licker
+cunt lips
+cuntasaurus rex
+cuntbag
+cuntbollock
+cuntelope
+cuntits
+cuntless
+cuntlick
+cuntlicker
+cuntlickers
+cuntness
+cunts
+cunty
+cuntz
+cuunt
+cyberfck
+cyberfcks
+cyberfuccs
+cyberfucks
+cyberfucs
+cyberfukk
+cyberfukks
+cyberfvcks
+d!ck
+d!cks
+d!ld0
+d!ldo
+d0ggy style
+d1ck
+d1ckhead
+d1ckheads
+d1cks
+d1cksucker
+d1cksukka
+d1ckz
+d1ldo
+dafuq
+dago
+damn
```

Comment:
I'm not sure about this one - most of the results that come with this search are not sensitive